src/interfaces/store-interfaces/state-model-interface.ts

```diff
@@ -26,6 +26,7 @@ export interface StateModelIF {
   originalIA: IncorporationFilingIF
   originalAlteration: AlterationFilingIF
   businessSnapshot: BusinessSnapshotIF
+  firmSnapshot: any
```

Comment:
Type? (If you have a good reason why not then please add a short comment explaining.)

Reply:
oh yea, same comment as above. I don't know exactly what the sp/gp model will look like, was going to update once we had some business records and a schema for the filing 😁